### PR TITLE
add test for static weight random remainder.

### PR DESCRIPTION
**Pull request by chaosi-zju**
**What type of PR is this?**

/kind failing-test

**What this PR does / why we need it**:

add test for static weight random remainder.

**Which issue(s) this PR fixes**:

Fixes part of #4220

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/scheduler/core/generic_scheduler_test.go [hunk 1]
```diff
@@ -82,8 +101,87 @@ func Test_genericScheduler_AssignReplicas(t *testing.T) {
...
+				if isScheduleResultEqual(got, want) {
+					return
+				}
+			}
+			t.Errorf("AssignReplicas() got = %v, wants %v", got, tt.wants)
+		})
+	}
+}
+
+func Test_genericScheduler_AssignReplicas_2(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (chaosi-zju) [reply pkg/scheduler/core/generic_scheduler_test.go:114]: the first test only schedule once, the second test schedule twice and changed schedule condition

[the remark this answers is not in the document]
Comment (chaosi-zju) [reply pkg/scheduler/core/generic_scheduler_test.go]: merged

**Comment by chaosi-zju** [comment]
@XiShanYongYe-Chang any more question? I think my this Test code is perfect (๑•̀ㅂ•́)و✧ ~

**Comment by XiShanYongYe-Chang** [comment]
OK
/assign

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm
/approve

**Pull request merged**